Dockerfile-mongo-seed

```diff
@@ -0,0 +1,13 @@
+FROM  mongo:3.2
+
+COPY dc-campaign-finance-mongodatabase.zip .
+
+# update packages and intall unzip
+RUN apt-get update
+RUN apt-get install unzip
```

Comment:
After you've done this, you'll want to do something like 

`apt-get remove --purge -y unzip && rm -rf /var/lib/apt/lists/*`

This will reduce the size of your image a bit, if you do it.